### PR TITLE
Upgrade Bootstrap and Bootswatch to v3.4.1, jQuery to v1.12.4

### PR DESCRIPTION
**Pull request by vedranmiletic**
Solar theme is unfortunately removed as it was never officially released with
v3.4.1 and instead came out only with Bootswatch v3.3.7 and v4.x.

Fixes issue #190.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
  History
 =========
 
+==========
+UNRELEASED
+==========
+* Upgrade Bootstrap and Bootswatch to v3.4.1. (`@vedranmiletic`_)
```
Comment (ryan-roemer) [inline HISTORY.rst]: Thanks for adding history -- form looks great! Your username needs to be defined blow at the bottom of the file like the others as build hits this issue:

```
WARNING: Unknown target name: "@vedranmiletic".
```

**Comment by vedranmiletic** [comment]
> You need on small fix in `HISTORY.rst` and then I'll merge and release!

Hopefully fixed.

**Comment by vedranmiletic** [comment]
Removed previously accidentally added `bootstrap.js.bak` file.

**Comment by ryan-roemer** [comment]
Release is delayed as my old `fabfile.py` release logic has finally bitten the dust as I need twine and installing that doesn't work with old fabric so far... I'll figure it out hopefully soon...

**Pull request merged**